tests/Integration/Schema/Types/InterfaceTest.php

```diff
@@ -341,11 +341,11 @@ interface HasPosts {
         $this->assertSame('PostPaginator', $result->json('data.__type.fields.0.type.ofType.name'));
     }
 
-    /** @return \Illuminate\Database\Eloquent\Collection<int, \Tests\Utils\Models\User|\Tests\Utils\Models\Team> */
-    public static function fetchResults(): EloquentCollection
+    /** @return \Illuminate\Support\Collection<int, \Tests\Utils\Models\User|\Tests\Utils\Models\Team> */
+    public static function fetchResults(): Collection
     {
-        /** @var \Illuminate\Database\Eloquent\Collection<int, \Tests\Utils\Models\User|\Tests\Utils\Models\Team> $results */
-        $results = new EloquentCollection();
+        /** @var \Illuminate\Support\Collection<int, \Tests\Utils\Models\User|\Tests\Utils\Models\Team> $results */
```

Comment:
Yeah, this is better. Eloquent collections are supposed to hold only a single model type.